h/search/config.py

```diff
@@ -105,6 +121,73 @@
     }
 }
 
+# Elasticsearch type mapping for annotations for ES 6.x and later.
+# This mapping does not include the legacy fields from the ES 1.x mapping.
+ES6_ANNOTATION_MAPPING = {
+    # Ignore unknown fields and do not add them to the mapping.
+    # This ensures that only fields included in the "properties" section
+    # here can be searched against.
+    'dynamic': False,
+
+    # Indexed (searchable) fields.
+    'properties': {
+        'authority': {'type': 'keyword'},
+        'created': {'type': 'date'},
+        'deleted': {'type': 'boolean'},
+        'nipsa': {'type': 'boolean'},
```

Comment:
These properties should be sorted alphabetically. Can you move the `nipsa` definition down accordingly?